jboss/container/wildfly/launch-config/os/added/launch/files.sh

```diff
@@ -19,7 +19,7 @@ function get_maven_artifacts_jar {
   incomplete=$2
   artifacts="$(xmllint --xpath "//*[local-name()='module']/*[local-name()='resources']/*[local-name()='artifact']/@name" $moduleFile 2>/dev/null)"
 
-  read -ra ARR <<< "$artifacts"
+  readarray ARR <<< "${artifacts}"
```

Comment:
I suspect this is a bug, as it returns only the first element for me from $artifacts. But it was unreliable using read -ar -- perhaps its a bug in fedora's bash. readarray works every time.